drivers/scheduler/k8s/k8s.go

```diff
@@ -70,6 +70,22 @@ const (
 	resizeSupportedAnnotationKey = "torpedo/resize-supported"
 )
 
+const (
+	//PortworxStorage portworx storage name
+	PortworxStorage = "portworx"
```

Comment:
PortworxStorage, CsiStorage, portworxStorage and csiStorage are not being used too

Reply:
Fixed.